changelog.md

```diff
@@ -1,5 +1,10 @@
 # illogical changelog
 
+## 1.5.0
+
+- Add simplify capability to evaluate the expression for all referred values
+  not present in the context except for a specified list of keys

```

Comment:
We may tweak this as well to be aligned with the new thermology (strict, optional).